recipes/memopair/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set version = "0.1.3" %}
+{% set name = "memopair" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SorenHeidelbach/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3b04dfbfcbc8cd47eb86eef3b31eab178d7d467afaf4d86197c91416e22e9088
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('memopair', max_pin="x.x") }}
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - cargo install -v --locked --no-track --root $PREFIX --path .
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+
+test:
+  commands:
+    - memopair --version
+
+about:
+  home: https://github.com/SorenHeidelbach/memopair/tree/master
+  license: MIT 
+  license_file:
+    - THIRDPARTY.yml  # this file is generated by cargo-bundle-licenses
+  summary: A package for identifying methylated motif pairs
+
+extra:
+  recipe-maintainers:
```

Please consider adding support for Linux/OSX ARM64:
```suggestion
  additional-platforms:
    - linux-aarch64
    - osx-arm64
  recipe-maintainers:
```